packages/blocks/src/code-block/components/lang-list.ts

```diff
@@ -98,6 +98,9 @@ export class LangList extends ShadowlessElement {
   @state()
   private _filterText = '';
 
+  @state()
+  private _currentSelectedIndex = -1;
```

Comment:
Would be better if this index can start from the hover position!

https://user-images.githubusercontent.com/7312949/230734987-a7558641-7ec1-4a4e-852c-328c9acd3beb.mov

Reply:
In fact, I don't think we need to do this because hovering and up and down keys are not the same behavior, and combining these two behaviors together would be a little bit strange. But if you want the editor acts like this, I will make some modification.

Reply:
Here's an example of a google search, which is a common behavior.

https://user-images.githubusercontent.com/15389209/230771543-bc67d373-2d58-46af-b41b-f55cfb1488e1.mov